CONTRIBUTING.md

```diff
@@ -189,6 +190,12 @@ Note: Not all keys are required, to run the app locally, however `MONGOHQ_URL` i
 
 You can leave the other keys as they are. Keep in mind if you want to use more services you'll have to get your own API keys for those services and edit those entries accordingly in the .env file.
 
+Next you should setup MailHog, a local SMTP mail server that will catch all the outgoing freeCodeCamp messages generated locally. How you start up MailHog is dependent upon your OS, but here's an example for MacOS with Brew.
+
+```bash
+brew services start mailhog
+```
+ 
```

Comment:
Please add additional info about opening up the localhost web view of the service available at 8025 port, or something on similar lines.